assets/js/romo/base.js

```diff
@@ -201,9 +201,21 @@ Romo.prototype.hide = function(elem) {
 
 Romo.prototype.offset = function(elem) {
   var rect = elem.getBoundingClientRect();
+  var scrollY = undefined;
+  var scrollX = undefined;
+  if (window.pageYOffset !== undefined)
+    scrollY = window.pageYOffset;
+    scrollX = window.pageXOffset;
+  } else if ((document.compatMode || "") === "CSS1Compat") {
```

Comment:
@jcredding I think we have other stuff in Romo that breaks on less than IE9.  Let's just use the page offset for simplicity.

I'd keep you notes about compatibility in the commit msg though.  That was good reference/context.